src/constraints.jl

```diff
@@ -152,14 +152,32 @@ function MOI.get(m::LinQuadOptimizer, ::Type{MOI.ConstraintIndex{F,S}}, name::St
     m.constraint_names_rev[name]::MOI.ConstraintIndex{F,S}
 end
 
+
+"""
+    CSRMatrix{T}
+
+Matrix given in compressed sparse row (CSR) format.
+
+It contains three vectors:
+ - `column_indices` is a vector of column numbers;
+ - `data` is a vector of corresponding nonzero values; and
+ - `row_pointers` is a vector pointing to the start of each row in
+    `column_indices` and `data`.
+
+The length of `row_pointers` is the number of rows in the matrix.
+"""
+struct CSRMatrix{T}
+    column_indices::Vector{Int}
+    data::Vector{T}
+    row_pointers::Vector{Int}
```

Comment:
The number of columns is missing and can't be inferred from `column_indices` (there could be empty columns).

Reply:
Hmm. This isn't really mean as a proper type that supports matrix operations. It's really just a collection of the three arrays to be passed to the solvers.

Reply:
It won't be long until it's more than a collection of three arrays, but this is fine for now. Maybe document this intent?